app/controllers/facility_statements_controller.rb

```diff
@@ -56,7 +54,7 @@ def send_statements
 
       @account_statements = {}
       to_statement.each do |account, order_details|
-        statement = Statement.create!(facility: current_facility, account_id: account.id, created_by: session_user.id)
+        statement = Statement.create!(facility: order_details.first.facility, account_id: account.id, created_by: session_user.id)
```

Comment:
Are the changes here in `send_statements` more related to the cross-facility statementing feature? If so, can we move them out of this PR?